openslides/core/static/js/core/start.js

```diff
@@ -46,6 +47,27 @@ angular.module('OpenSlidesApp.core.start', [])
                 this.bootup();
             },
         };
+
+        // We need to 'ping' the server with a get request to whoami, because then we can decide,
+        // if the server is down or respond with a 403 (this cannot be differentiated with websockets)
+        autoupdate.registerRetryConnectCallback(function () {
+            return $q(function (resolve) {
+                $http.get('/users/whoami').then(function (success) {
+                    if (success.data.user_id === null && !success.data.guest_enabled) {
+                        OpenSlides.shutdown();
```

Comment:
Why not just OpenSlides.reboot()? Maybe we should rewrite bootup, shutdown and reboot so that they return each a promise.